specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -1341,14 +1291,657 @@
             "type": "string"
           },
           {
-            "name": "appSettings",
-            "in": "body",
-            "description": "The dictionary containing the static site app settings to update.",
-            "required": true,
-            "schema": {
+            "name": "environmentName",
+            "in": "path",
+            "description": "The stage site identifier.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/StringDictionary"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get function app settings of a static site build": {
+            "$ref": "./examples/ListStaticSiteBuildFunctionAppSettings.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/builds/{environmentName}/showDatabaseConnections": {
+      "post": {
+        "tags": [
+          "StaticSites"
+        ],
+        "summary": "Returns details of database connections for a static site build",
+        "operationId": "StaticSites_GetBuildDatabaseConnectionsWithDetails",
```

Comment:
>StaticSites_GetBuildDatabaseConnectionsWithDetails [](http://example.com/codeflow?start=24&length=50)

nitpick, why put 'WithDetails' in the name? Aren't you just returning all the normal resource properties? Where does the WithoutDetails scenario come in?

Reply:
This is mainly to distinguish the POST (which provides the connection string) from the regular GET (which does not provide the connection string). The `WithDetails` is to be a bit more generic for future-proofing if additional properties which should be in the POST response but not GET appear.